uqcsbot/text.py

```diff
@@ -273,6 +286,44 @@ async def zalgo_command(self, interaction: discord.Interaction, text: str):
 
         await interaction.response.send_message(self.zalgo_common(text))
 
+    def rot_13_cipher(self, text: str) -> str:
```

Comment:
I'm a bit surprised that modulo arithmetic wasn't used, but this implementation is quite readable.

Reply:
Modular arithmetic means having to normalise to "a"=0, then do the modulo, then denormalise, which to me seems messier than this implementation
```python
def rot_13_cipher(self, text: str) -> str:
    result = ""
    for c in text:
        if "a" <= c <= "z":
            result += chr((ord(c) - ord("a") + 13) % 26 + ord("a"))
        elif "A" <= c <= "Z":
            result += chr((ord(c) - ord("A") + 13) % 26 + ord("A"))
        else:
            result += c
    return result
```
IIRC, I used modular arithmetic for the original `!caesar` command, as that had a variable rotation.